### ISSUE TITLE
SVG images do not display on kiwix-serve/chrome extension

### ISSUE THREAD

**Issue by satyamtg** [openzim/openedx#101]
In the zim created by [this](https://farm.openzim.org/pipeline/5f2ce0f92a0cfbdda0a33fcb) run, SVGs do not display correctly. However, the paths are good and files are present. Also, testing the same on Kiwix Android, the SVGs display flawlessly. Here's a screenshot of what it looks like on kiwix-serve -
![Screenshot from 2020-08-07 15-18-17](https://user-images.githubusercontent.com/17774888/89639522-54bf3800-d8cb-11ea-8a1f-2abef0d7c267.png)

On the previous run, i.e. this one, SVGs used to display properly. However, that one used the zimwriterfs. The newer one uses pylibzim.

**Comment by rgaudin** [comment]
The difference that could make is regarding the MIME type…

**Comment by satyamtg** [comment]
> The difference that could make is regarding the MIME type…

Indeed. Looking at the content-type header in the previous ZIM (using zimwriterfs), and this one (using pylibzim), we can clearly see that previously it was `image/svg+xml` whereas now it's `image/svg` (which seems to be invalid). This might be an upstream issue.

**Comment by satyamtg** [comment]
After looking into this, I have found the root cause of the problem. Its not an error with this scraper but has something to do with docker. Actually, in the docker container, we run on a debian image and in it, magic identification actually reports `image/svg`, and that's inherent due to a different `magic.mgc` database on different platforms. 

For fixing this, I think we can go one of the following ways -
- Fix this in scraperlib and convert `image/svg` to `image/svg+xml` (we can create a mapping for such mimetypes, if something else pops up in the future, we just put it in the mapping)
- Fix this in docker (Tried updating and installing things, but out of luck. I think the only option here is to build magic database from source, and it may have its own issues)

I would suggest we transfer this issue to scraperlib and fix there (as mentioned above). I also tried a pure python libmagic implementation, called puremagic but its results seem to be out of this world (at least for svg images).

BTW, @rgaudin do you know how does zimwriterfs deal with it?

**Comment by rgaudin** [comment]
We already have such a mapping I believe so it would be just adding an
entry. That sounds like the best way to go.
